xapian-applications/omega/templates/query

```diff
@@ -104,17 +109,19 @@ $html{$date{$field{modtime},%Y-%m-%d}}</time></span>
 </noscript>
 }
 $if{$field{language},Language: <b>$html{$field{language}}</b>}
-$if{$field{size},<span title="$html{$field{size}} bytes">Size: <b>$html{$filesize{$field{size}}}</b></span>}
+$if{$field{size},<div title="$html{$field{size}} bytes">Size: <b>$html{$filesize{$field{size}}}</b></div>}
```

Comment:
True, it was added to reuse the list-item-meta, forgot to add the classs. Will revert it back to span